.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,30 @@
+name: Unit tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
I don't think this way it will run for PRs like that, what about using just `on: [push]`?

Reply:
Yes, you are right! Now it is fixed and the first tests have been executed.